_scripts/travis-deploy.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Deploy _site to GCS bucket, but only if this is a master (gh-pages) build.
+set -e
+
+BUCKET="gs://weasel-webrtc"
+dir=$(dirname $0)
+
+[ "$TRAVIS_PULL_REQUEST" != "false" ] && exit 0
+[ "$TRAVIS_SECURE_ENV_VARS" != "true" ] && exit 0
+[ "$TRAVIS_BRANCH" != "gh-pages" ] && exit 0
+
+openssl aes-256-cbc -d \
```

Comment:
I mean, look for previous runs on travis for travis-ci branch. You'll see that it works.